requirements-azure.txt

```diff
@@ -1,3 +1,3 @@
 azure-datalake-store >= 0.0.30
-azure-storage-blob
-requests >= 2.21.0
+azure-storage-blob == 2.1.0
```

Comment:
Thanks -- someone needs to dig into why the incompatible change in a separate PR . But pinning for now is probably the right change.